docs/pages/setup/reference/config.mdx

```diff
@@ -483,6 +483,10 @@ auth_service:
     # See "TLS Routing" in Architecture section for additional information.
     proxy_listener_mode: multiplex
 
+    # Sends the host CAs of all clusters to SSH clients logging in when enabled,
+    # instead of just the host CA for the current cluster. Defaults to false.
```

Comment:
Do we need to include something here that explains why this is an option that a user would want to enable ? This description alone feels like it requires an in-depth understanding of Teleport to know why it is useful.

Reply:
Agree with the above. Also, does "SSH clients" refer to `tsh` only, or using the OpensSSH with `tsh config`?